pkg/resources/account_password_policy_attachment.go

```diff
@@ -0,0 +1,85 @@
+package resources
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/helpers"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var accountPasswordPolicyAttachmentSchema = map[string]*schema.Schema{
```

Comment:
should this be renamed to "password_policy_account_attachment"?

Reply:
password policies can be attached to either accounts or users, so I figured we'd have a separate resource later user_password_policy_attachment